l10n_es_aeat_mod190/models/account_invoice.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _name = 'account.invoice'
+    _inherit = ['account.invoice', 'l10n.es.mod190.mixin']
+
+    @api.model
+    def line_get_convert(self, line, part):
+        """Copy from invoice to move lines"""
+        res = super(AccountInvoice, self).line_get_convert(line, part)
```

Comment:
Why doing this twice, one here and the other in `invoice_line_move_line_get`?

Reply:
La aeat_perception_key y subkey solo deben ponerse en las líneas que vienen de linea de factura, por lo tanto, no se replicarà en impuestos o la de pago. Por eso se hizo de esta manera.